.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         python-version: [  '3.8', '3.9', '3.10', '3.11']
-        myst-parser-version: [ '<0.18.0', '>=0.18.0,<0.19', '>=0.19.0,<1.0', '>=1.0.0,<2', '>=2.0.0,<3']
```

Comment:
Seems fine to me, I'm thinking there may be a reason for the old version here but downstream of sphinxcontrib-opendataservices-jsonschema are going to need to update. You'll need to make a release and can add that in the notes, if it's very important someone can add it back.

Reply:
Looks like all the repos depending on this, have the version pinned, so hopefully will be not problems.